meetings/metafiles/utils.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+from django.apps import AppConfig
+from meetings.utils import OsfOauth2AdapterConfig
+
+
+class OsfFileStorageUrls(AppConfig):
+    BASE_URL = '{}oauth2/{}'.format(
```

Comment:
Why are you setting what should either be in the `defaults.py` or `local.py` here?